tests/test_bulk_tx_badack/src/main.xc

```diff
@@ -13,58 +12,27 @@
 #define EP_COUNT_OUT          (6)
 #define EP_COUNT_IN           (6)
 
-/* Endpoint type tables */
+#include "shared.h"
+
 XUD_EpType epTypeTableOut[EP_COUNT_OUT] = {XUD_EPTYPE_CTL,
                                                 XUD_EPTYPE_BUL,
```

Comment:
It appears that this line has a different indentation to the ones which follow.  Is that intentional?

Reply:
Fixed.